templates/articles/proposal.html

```diff
@@ -12,16 +15,41 @@
         <option value=es>Spanish</option>
       </select>
     </label>
-    <label for=field_topic>topic *</label><input name=topic id=field_topic required>
-    <label for=field_angle>angle</label><input name=angle id=field_angle>
-    <label for=field_format>format</label><input name=format id=field_format>
-    <label for=field_additional>additional</label><textarea name=additional id=field_additional></textarea>
-    <label for=field_name>name *</label><input name=name id=field_name required>
-    <label for=field_email>contact *</label><input type=email name=email id=field_email required>
-    <label for=field_city>city</label><input name=city id=field_city>
+    <label for=field_name>Name *</label><input name=name id=field_name required>
+    <label for=field_email>Contact *</label><input type=email name=email id=field_email required>
+    <label for=field_city>City</label><input name=city id=field_city>
+    <label for=field_topic>Topic in a sentence *<p>ex: Digital nomads in Lisbon</p></label><input name=topic id=field_topic required>
+    <label for=field_angle>Pitch your story<p>ex: I want to meet young people/ participants of the three major digital nomad communities/ events in Lisbon and use their personal stories as an insight to describe the digital nomads landscape in Portugal</p></label><input name=angle id=field_angle>
+    <hr>
+    <label for=media-type>Media
```

Comment:
Changing these forms fields will require to adapt the backend (views and tests).
David or myself should pair with you to complete this PR.

Reply:
ok, this might not be the final-final version, so maybe we do this later

Reply:
Tests are done. Next step may be merging.